frontend/src/App.js

```diff
@@ -121,6 +122,12 @@ const RouteList = () => {
           path="master-data"
           element={<Private element={MasterData} alias="master-data" />}
         />
+        <Route
+          path="data/upload-administration-data"
```

Comment:
`master-data`

Reply:
instead of the "/upload-administration-data"  we need to replace to master-data right?

Reply:
no, `master-data/upload-administration-data`